src/ui/assets/AssetsListItem.js

```diff
@@ -39,7 +43,10 @@ const AssetsListItem = ({ asset, domain, onEditClicked }) => {
           <MenuItem className="" onClick={onEditClickHandle}>
             <FormattedMessage id="cms.label.edit" defaultMessage="Edit" />
           </MenuItem>
-          <MenuItem className="" onClick={() => {}}>
+          <MenuItem className="" onClick={() => window.open(downloadUrl)}>
```

Comment:
separate the function in `onClick` handle like what you did on `onEditClickHandle` and `onClickDeleteHandle`